src/stories/checkbox/unchecked.stories.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react"
```

Comment:
I don't think there's much value in duplicating all stories just for the sake of showing them in checked / unchecked state. one can do that by just checking the checkbox in the other file... also, there are variations for checked/unchecked already above

what do you think about dropping these? maybe you see value I'm missing....

Reply:
I think you are right.  As it is right now the box is checked and unchecked on click and the` checked `args in the control pannel does not change accordingly. Will try to fix that.